scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -73,10 +76,19 @@ workflow GvsCreateFilterSet {
         File? service_account_json
 
         Int? SNP_VQSR_machine_mem_gb
-        Int? SNP_VQSR_downsampleFactor = 1
+        Int SNP_VQSR_downsampleFactor = 1
 
         Int? INDEL_VQSR_machine_mem_gb
 
+        Int snps_variant_recalibration_threshold = 20000
+    }
+
+    call GetNumSamples {
+        input:
+            fq_sample_table = fq_sample_table,
+            service_account_json = service_account_json,
+            project_id = query_project,
+            dataset_name = default_dataset
```

Comment:
I think this is unused (given the fq sample table name and query project)

Reply:
you are correct — good catch!